### ISSUE TITLE
Global Error Handler is never executed

### ISSUE THREAD

**Issue by Frankistan** [AndrewKeig/express-validation#67]
According to the documentation here:  https://github.com/AndrewKeig/express-validation#distinguish-errors-from-validationerrors , the error handler is never reached although I got the validation errors in JSON format in the console but not in Postman

How could I get the validation errors in JSON format using Postman?

**Comment by tesarm** [comment]
can you send us your code? It works fine for me .

**Comment by tesarm** [comment]
you are probably missing something like this:

```

app.use((err, req, res, next) => 
    res.status(err.status).json({
        message: err.isPublic ? err.message : httpStatus[err.status],
        stack: config.env === 'development' ? err.stack : {}
    })
);


```

**Comment by msudgh** [comment]
It doesn't work. never reach the specified middleware.
Maybe, you throw the error in the wrong way.

**Comment by AndrewKeig** [comment]
Closing, released 3.0
